test/mock_test.py

```diff
@@ -42,6 +43,31 @@ def test_with(self):
         with t.open('r') as b:
             self.assertEqual(list(b), ['bar'])
 
+    def test_bytes(self):
+        t = MockTarget("foo", format=Nop)
+        with t.open('wb') as b:
+            b.write(b"bar")
+
+        with t.open('rb') as b:
+            self.assertEqual(list(b), [b'bar'])
+
+    def test_default_mode_value(self):
+        t = MockTarget("foo")
+        with t.open('w') as b:
+            b.write("bar")
+
+        with t.open() as b:
+            self.assertEqual(list(b), ['bar'])
+
+    def test_mode_none_error(self):
+        t = MockTarget("foo")
+        try:
+            with t.open(None) as b:
+                b.write("bar")
+        except TypeError as error:
+            self.assertIsNotNone(error)
+            self.assertIn("'NoneType' object", str(error))
```

Comment:
can this test use [`assertRaises`](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises)?

Reply:
Yes